lib/audit.js

```diff
@@ -13,7 +13,7 @@ const audit = async args => {
   const fix = args[0] === 'fix'
   const result = await arb.audit({ fix })
   if (fix) {
-    reifyOutput(arb)
+    await reifyOutput(arb)
```

Comment:
`reifyOutput` should not be returning a promise any more, I think?  I seem to recall that was because `npm-audit-report` was returning a promise unnecessarily, but the v2 version doesn't do that.

Reply:
UPDATE: I see, it's because `libnpmfund.read()` is async.